src/core/include/openvino/op/constant.hpp

```diff
@@ -12,16 +12,19 @@
 #    define WAS_OV_LIBRARY_DEFINED_CONSTANT
 #endif
 
-#include "ngraph/util.hpp"
 #include "openvino/core/rtti.hpp"
 
 #ifdef WAS_OV_LIBRARY_DEFINED_CONSTANT
 #    undef IN_OV_COMPONENT
 #    undef WAS_OV_LIBRARY_DEFINED_CONSTANT
 #endif
```

Comment:
we can remove these macroses in a separate PR

Reply:
Removed in this one.